core/src/window_service.rs

```diff
@@ -167,6 +167,13 @@ fn run_check_duplicate(
             }
         };
 
+        if root_bank
+            .feature_set
+            .is_active(&solana_feature_set::secp256k1_program_enabled::id())
+        {
+            return Ok(());
+        }
+
         // Propagate duplicate proof through gossip
         cluster_info.push_duplicate_shred(&shred1, &shred2)?;
```

Comment:
This also stops sending duplicate proofs through gossip, which takes care of the other `duplicate_slots_sender` on gossip proof receival from honest nodes.

we should have another PR to drop duplicate shred proof and epoch slots frozen messages in gossip itself when the feature flag is active